types/liferay/index.d.ts

```diff
@@ -0,0 +1,3 @@
+/// <reference path="liferay.d.ts" />
+
+export = Liferay;
```

Comment:
I think, if the intent of the package is to be used as global constants, as the types indicate. you shouldn't actually export anything.
Unless users of the underlying API/source code can actually import something at run time.